rotkehlchen/chain/ethereum/graph.py

```diff
@@ -52,9 +54,9 @@ class Graph():
     def __init__(self, url: str) -> None:
         """
         - May raise requests.RequestException if there is a problem connecting to the subgraph"""
-        transport = RequestsHTTPTransport(url=url, retries=QUERY_RETRY_TIMES)
+        transport = RequestsHTTPTransport(url=url)
         try:
-            self.client = Client(transport=transport, fetch_schema_from_transport=False)
+            self.client = Client(transport=transport)
```

Comment:
Why remove `fetch_schema_from_transport` here? Even if `False` is the default, if they change that default in a next version the bug will appear again. Always prefer explicitness in such cases.